### PR TITLE
Add emoji hint functionality

### PR DESCRIPTION
**Pull request by canadianveggie**
Give our power users a way to discover unlocked emojis.

Accessed by pressing `?` on the #collection page or by pressing the `Need a Hint?` button at the bottom.
The button only shows if you have at least 100 emojis in your colleciton.

Gives a new hint every 1 hour.

Screenshot 2023-11-07 at 4 36 35 PM

### CODE REVIEW

**Review by wachunga — APPROVED** [review]
Cool feature! I think the 50 is kind of arbitrary but agree you need some kind of minimum

**Comment by canadianveggie** [comment]
> Cool feature! I think the 50 is kind of arbitrary but agree you need some kind of minimum

This is mostly for power users and intended to address feedback like:
https://twitter.com/TheVidaBlues/status/1717547919153225826

When your collection is small it's easy to unlock new emojis. Should we go higher? 100, 200?
What feels right to you?

**Comment by wachunga** [comment]
> > Cool feature! I think the 50 is kind of arbitrary but agree you need some kind of minimum
> 
> This is mostly for power users and intended to address feedback like: https://twitter.com/TheVidaBlues/status/1717547919153225826
> 
> When your collection is small it's easy to unlock new emojis. We can set the minimum higher. 100 or 200? What feels right to you?

I think 50 works fine

File: index.html [hunk 1]
```diff
@@ -276,6 +276,9 @@ <h1>Collection</h1>
           <span x-text="Math.floor(Object.keys(emojis).length / 100)  * 100"></span>
           emojis in total...
         </p>
+        <div class="hint-area" x-show="unlockedEmojis.length > 50" x-cloak>
```
Comment (canadianveggie) [inline index.html:279]: Let's go with a nicer round number
```suggestion
        <div class="hint-area" x-show="unlockedEmojis.length >= 100" x-cloak>
```

**Pull request merged**